site/src/uiuisms.rs

```diff
@@ -117,6 +117,10 @@ uiuisms!(
     "/↧ [1 4 5 3 2]",
     /// Sort an array
     "⊏⌂. [1 4 5 3 2]",
+    /// Get the average of an array
+    "÷⇉⧻/+ [1 2 3]",
+    // Get the index mask of a bit array
+    "▽∶⇡⧻. [1 0 0 1]",
```

Comment:
This is actually present further down as "Find the indices of all 1s".
Maybe the description needs to be reworded to make it easier to search for?

Reply:
Ah you're correct! My bad.